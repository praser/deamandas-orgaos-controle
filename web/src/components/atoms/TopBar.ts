import styled from 'styled-components';

const TopBar = styled.div`
  /* Layout */
  align-items: center;
  box-sizing: border-box;
  display: flex;
  flex-direction: row;
  flex-wrap: nowrap;
  height: 70px;
  justify-content: flex-start;
  margin-bottom: 24px;
  padding-bottom: 8px;
  padding-left: 16px;
  padding-right: 16px;
  padding-top: 8px;
  position: relative;
  width: 100%;

  /* Appearance */
  color: rgb(133, 135, 150);
  background-color: rgb(255, 255, 255);
  box-shadow: rgba(58, 59, 69, 0.15) 0px 2.4px 28px 0px;
`;

export default TopBar;
