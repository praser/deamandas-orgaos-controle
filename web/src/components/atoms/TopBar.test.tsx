import React from 'react';
import { render, screen } from '@testing-library/react';
import TopBar from './TopBar';

describe('TopBar atomic component', () => {
  let topBar: HTMLElement;

  beforeEach(() => {
    render(<TopBar data-testid="top-bar" />);
    topBar = screen.getByTestId('top-bar');
  });

  it('is expected to match the snapshot', () => {
    expect(topBar).toMatchSnapshot();
  });
});
