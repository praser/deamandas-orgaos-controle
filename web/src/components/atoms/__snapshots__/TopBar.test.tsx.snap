// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TopBar atomic component is expected to match the snapshot 1`] = `
<div
  class="sc-bdvvaa jxZMhP"
  data-testid="top-bar"
/>
`;
